__tests__/smoke.test.js

```diff
@@ -1,6 +1,6 @@
 const PPTX = require('../index.js');
 describe('test essentials', () => {
     test('true', () => {
-        expect(Object.keys(PPTX)).toEqual(['Presentation', 'Slide']);
+        expect(Object.keys(PPTX)).toEqual(['Presentation', 'Slide', 'ShapeTypes']);
```

Comment:
Maybe just `Shapes`?

Reply:
Sounds to similar to the "Shape" class though...